plasmapy/optional_deps.py

```diff
@@ -0,0 +1,50 @@
+""" Useful error messages for optional dependencies that aren't found. """
+from typing import Optional
+
+def _template(pkgname: str,
```

Comment:
Perhaps rename `_template` to something more specific, like `_import_error_template`?